### PR TITLE
Debug vector add Windows CI failure.

### PR DESCRIPTION
**Pull request by sgundabo**
fixed missing \<CR\> in the kernel

### CODE REVIEW

**Review by junliume — APPROVED** [review]

**Review by apwojcik — APPROVED** [review]

File: test/gtest/vec_add.cpp [hunk 1]
```diff
@@ -114,16 +114,20 @@ struct VecAddTest : public ::testing::TestWithParam<VecAddTestCase>
 
         std::string params = options.GenerateFor(miopen::kbp::OpenCL{});
 
-        uint totalElements  = vec_size;
```
Comment (amberhassaan) [inline test/gtest/vec_add.cpp:117]: I hope you have learned by now that `uint` is not the correct nomenclature :).

File: test/gtest/vec_add.cpp [hunk 2]
```diff
@@ -147,9 +151,9 @@ struct VecAddTest : public ::testing::TestWithParam<VecAddTestCase>
 
         std::string params = options.GenerateFor(miopen::kbp::HIP{});
 
-        uint totalElements  = vec_size;
-        int threadsPerBlock = threads_per_block;
-        int blocksPerGrid   = (totalElements + threadsPerBlock - 1) / threadsPerBlock;
+        size_t totalElements = vec_size;
```
Comment (amberhassaan) [inline test/gtest/vec_add.cpp]: Several minor but important issues to address: 
* `vecadd_config` and `vec_size, threads_per_block` are duplicating the same info. Keep `vecadd_config` and get rid of the other two members.

File: test/gtest/vec_add.cpp [hunk 3]
```diff
@@ -147,9 +151,9 @@ struct VecAddTest : public ::testing::TestWithParam<VecAddTestCase>
...
+        int threadsPerBlock  = threads_per_block;
```
Comment (amberhassaan) [inline test/gtest/vec_add.cpp]: Do not convert `size_t` to `int`. That's a `narrowing` conversion. Either keep everything as `int or unsigned int` or keep everything as `size_t`.

**Review by amberhassaan — APPROVED** [review]
Please address the comments before merging.

**Comment by sgundabo** [comment]
@junliume Please merge this when you get a chance.

**Review by junliume — APPROVED** [review]

**Pull request merged**